engine/apps/schedules/ical_utils.py

```diff
@@ -358,14 +358,14 @@ def list_users_to_notify_from_ical_for_period(
     schedule: "OnCallSchedule",
     start_datetime: datetime.datetime,
     end_datetime: datetime.datetime,
-) -> typing.List["User"]:
+) -> typing.Sequence["User"]:
     users_found_in_ical: typing.Sequence["User"] = []
```

Comment:
it seems like `users_found_in_ical` defined here is never used, since the variable is overwritten down on line 368?

Reply:
yeah 🤔 I'll remove this